contracts/Comet.sol

```diff
@@ -695,19 +693,15 @@ contract Comet is CometMainInterface {
     /**
      * @dev Multiply a `fromScale` quantity by a price, returning a common price quantity
      */
-    function mulPrice(uint128 n, uint128 price, uint64 fromScale) internal pure returns (uint) {
-        unchecked {
-            return uint256(n) * price / fromScale;
-        }
+    function mulPrice(uint n, uint128 price, uint64 fromScale) internal pure returns (uint) {
+        return n * price / fromScale;
     }
 
     /**
      * @dev Multiply a signed `fromScale` quantity by a price, returning a common price quantity
      */
-    function signedMulPrice(int128 n, uint128 price, uint64 fromScale) internal pure returns (int) {
-        unchecked {
-            return n * signed256(price) / signed256(fromScale);
-        }
+    function signedMulPrice(int n, uint128 price, uint64 fromScale) internal pure returns (int) {
```

Comment:
Hm, probably just for the `unchecked` math too here...good point, get rid of that too?

Reply:
I think we should maybe just return `uint256` instead of `uint128` for `getPrice`, since all the prices are eventually converted to `uint256/int256` when used.

In `getPrice`, we have the following check:

`if (price <= 0 || price > type(int128).max) revert BadPrice();`

I don't think the latter check `price > type(int128).max` makes much sense.

Reply:
Yea, thats what I meant 'get rid of'

Reply:
Updated